bean-consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/BeanConsistencyChecker.java

```diff
@@ -76,10 +88,17 @@ public boolean consistent(final Object o1, final Object o2) {
                 }
 
                 // compare values
-                if (!Objects.equals(o1Value, o2Value)) {
-                    if (logger.isDebugEnabled())
-                        logger.debug("Object 1 and Object 2 have "+field.getName()+" inconsistent");
-                    return false;
+                if (o1Value instanceof Timestamp && o2Value instanceof Date || o1Value instanceof Date && o2Value instanceof Time) {
```

Comment:
Is this not enough?

if ( o1Value instanceof java.util.Date and o2Value instanceof java.util.Date) ...